api/views.py

```diff
@@ -291,7 +291,18 @@ def handle_post(self, request, *args, **kwargs):
         username = body['username']
         password = body['password']
 
-        user = authenticate(username=username, password=password)
+        #allowing different lower/uppercase lettered usernames:
+        caseSensitiveUsername = username
+        try:
+            findUser = User._default_manager.get(username__iexact=username)
+        except User.DoesNotExist:
+            findUser = None
+        if findUser is not None:
+            caseSensitiveUsername = findUser
```

Comment:
It's more straightforward here to just say 

`try: case_sensitive_username = User.objects.get(username__iexact=username)`
`except User.DoesNotExist: case_sensitive_username = None`

Reply:
It would be the safest if you could do the change please. I just used a stackoverflow-found solution (this is the reason of the different variable format also).

Reply:
This is already in master branch.

Reply:
You could just open another PR?

Reply:
I created https://github.com/IFRCGo/go-api/pull/308 for this.